compiler/lsp-tests/src/Main.hs

```diff
@@ -416,6 +417,44 @@ scenarioTests run = testGroup "scenarios"
           closeDoc main'
     ]
 
+executeCommandTests :: (forall a. Session a -> IO a) -> (Session () -> IO ()) -> TestTree
+executeCommandTests run _ = testGroup "execute command"
+    [ testCase "execute commands" $ run $ do
+        main' <- openDoc' "Main.daml" damlId $ T.unlines
+            [ "daml 1.2"
+            , "module Coin where"
+            , "template Coin"
+            , "  with"
+            , "    owner : Party"
+            , "  where"
+            , "    signatory owner"
+            , "    controller owner can"
+            , "      Delete : ()"
+            , "        do return ()"
+            ]
+        Just escapedFp <- pure $ uriToFilePath (main' ^. uri)
+        actualDotString :: ExecuteCommandResponse <- LSP.request WorkspaceExecuteCommand $ ExecuteCommandParams
+           "daml/damlVisualize"  (Just (List [Aeson.String $ T.pack escapedFp]))
+        let expectedDotString = "digraph G {\ncompound=true;\nrankdir=LR;\nsubgraph cluster_Coin{\nn0[label=Create][color=green]; \nn1[label=Archive][color=red]; \nn2[label=Delete][color=red]; \nlabel=Coin;color=blue\n}\n}\n"
```

Comment:
This looks more like golden test. Let me know if there is a better way to do this.